regression/pages/studio/studio_textbooks.py

```diff
@@ -69,3 +70,14 @@ def delete_all_textbooks(self):
         """
         while self.get_textbook_count() > 0:
             self.click_delete_button()
+
+    def click_view_live_textbook(self):
+        """
+        Clicks View Live button on the first textbook available
+        """
+        button = self.q(css='.view').results[0]
+        # This button is hidden, hovering on it makes it visible
```

Comment:
That's ok. Once someone tabs to it, can they hit enter and un-hide it? That would help folks that need to use a keyboard, and it might imply that folks with a screen reader can also read the text.